addons/stock_picking_batch/wizard/stock_package_destination.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ChooseDestinationLocationBatch(models.TransientModel):
+    _name = 'stock.package.destination.batch'
```

Comment:
did you investigate if it was possible to re-use or extend the original stock package destination wizard ?

Reply:
Revisited it and extended original stock package destination wizard. It's maybe a little hacky, but it's less code + no duplicate template. Couldn't do the same with the `stock.picking put_in_pack` logic though